### PR TITLE
[T3A4][T15-A4]AngShiYa

### PR DESCRIPTION
**Pull request by AngShiYa**

### CODE REVIEW

**Comment by AngShiYa** [comment]
Ready for review

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -107,6 +107,9 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
+            if (!storage.ifFileExist()) {
+                ui.showFileNotFoundMessge();
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/Main.java]: Good catch. You can use the `ui` object only from the main class.

**Comment by nishantbudhdev** [comment]
@AngShiYa 

* The error handling was expected in the `save` method (in StorageFile class) and not directly in the `executeCommand` method (in Main class).
* You need to throw the error till the Main class, where you should call the function you created in the Textui class to print the error message.

File: src/seedu/addressbook/storage/StorageFile.java [hunk 2]
```diff
@@ -3,12 +3,14 @@
 import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.data.exception.IllegalValueException;
 import seedu.addressbook.storage.jaxb.AdaptedAddressBook;
+import seedu.addressbook.ui.TextUi;
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/storage/StorageFile.java]: Is this needed here ?

**Pull request closed without merging**